src/transport.ts

```diff
@@ -210,19 +210,12 @@ export class WebRTCTransport implements Transport {
       throw invalidArgument('no local certificate')
     }
 
-    const localCert = pc.getConfiguration().certificates?.at(0)
-
-    if (localCert === undefined || localCert.getFingerprints().length === 0) {
-      throw invalidArgument('no fingerprint on local certificate')
-    }
```

Comment:
Not sure how this is handled in the js-libp2p codebase in general. Should we still use `localCert.getFingerprints` in case the API is available (e.g. on Chrome), or should we always use the hack through the SDP?

Reply:
I think I would prefer trying to use the best option and falling back on the hack. But not blocking this.

Reply:
I agree with Marco here, but it would be good to know (debug logs) which option was used if we're going to have different paths.

Reply:
@SgtPooki I've added the fallback to fetching from the SDP as requested.